app/views/layouts/application.html.erb

```diff
@@ -9,7 +11,46 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
+  <body class="m-0">
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <%= link_to root_path do  %>
+        <span class="navbar-brand text-info font-weight-bold d-block">Private Events</span>
+      <% end %>
+
+      <div class="collapse navbar-collapse" id="navbarNav">
+      </div>
+
+      <div>
+        <% if current_user %>
+          <button class="border-0 bg-info text-light px-3 py-2 rounded mr-2">
+            <%= current_user.username %>
+          </button>
+          <span>
+            <%= link_to "Logout", logout_path, method: :delete, class:"text-info" %>
+          </span>
+        <% else cred_page? %>
+          <%= link_to login_path do  %>
+            <button class="border-0 bg-info text-light px-3 py-2 rounded">Sign in</button>
+          <% end %>
+          <%= link_to signup_path do  %>
+            <button class="border-0 bg-info text-light px-3 py-2 rounded">Sign up</button>
+          <% end %>
+        <% end %>
+      </div>
+    </nav>
+    <div class="container" id="notice">
+      <% if flash[:notice] %>
+         <div>
+           <p><%= notice %></p>
+         </div>
+       <% end %>
+       <% if flash[:alert] %>
+         <div>
+           <p><%= alert %></p>
+         </div>
+       <% end %>
```

Comment:
- [ ] these if statements should be put in helper methods instead of having them straight in the view